presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AggregationNode.java

```diff
@@ -71,6 +71,26 @@ public boolean isOutputPartial()
         {
             return outputPartial;
         }
+
+        public static Step partialOutput(Step step)
```

Comment:
it is used only there, but it fits much better here, since it manipulates on those `boolean inputRaw, boolean outputPartial` values. 

Those methods takes some `Step` and returns matching step with "inputRaw` or `outputPartial` flag set to true. Maybe this methods should be named `setPartialOutput` and `setPartialInput`? Or `markPartialOutput`? Or `convertToPartialOutput`?